src/app/beer_garden/app.py

```diff
@@ -93,6 +95,22 @@ def initialize(self):
             action=beer_garden.router.forward, name="forwarder"
         )
 
+        BaseManager.register(
+            "PluginManager",
+            callable=partial(
+                PluginManager,
+                plugin_dir=config.get("plugin.local.directory"),
+                log_dir=config.get("plugin.local.log_directory"),
+                connection_info=config.get("entry.http"),
+                username=config.get("plugin.local.auth.username"),
+                password=config.get("plugin.local.auth.password"),
+            ),
+        )
+
+        data_manager = BaseManager()
+        data_manager.start()
+        beer_garden.local_plugins.manager.lpm_proxy = data_manager.PluginManager()
```

Comment:
When initializing the Base Manager, why didn't you expose it with an address object? Because if you did, then we could have any where in our code initialize this class?

```python
data_manager = BaseManager(address=('', 50000)
```

Reply:
Yeah, I looked at that and it should work. I wasn't able to get it working yet, though.